ceph/utils.py

```diff
@@ -1160,9 +1161,14 @@ def set_container_info(ceph_cluster, config, use_cdn, containerized):
         config["use_cdn"] = True
         ansi_config["ceph_origin"] = "repository"
         ansi_config["ceph_repository_type"] = "cdn"
+    elif containerized:
+        ansi_config["ceph_docker_registry"] = config.get("ceph_docker_registry")
+        ansi_config["ceph_docker_image"] = config.get("ceph_docker_image")
+        ansi_config["ceph_docker_image_tag"] = config.get("ceph_docker_image_tag")
     else:
-        if containerized:
-            ansi_config["ceph_docker_registry"] = config.get("ceph_docker_registry")
-            ansi_config["ceph_docker_image"] = config.get("ceph_docker_image")
-            ansi_config["ceph_docker_image_tag"] = config.get("ceph_docker_image_tag")
+        container = get_latest_container(build)
```

Comment:
I think these details would be in config already.

Reply:
we may not need this else part, because it could be,
1) CDN - for this we do not want to update any image, if CDN container image is default.
2) Containerized 
3) else it would be RPM only.